### ISSUE TITLE
psad --Status flushes ips

### ISSUE THREAD

**Issue by hurricanehrndz** [mrash/psad#40]
Whenever I execute psad --Status it writes to socket the ips, removing them from the block. Why?

**Comment by hurricanehrndz** [comment]
Okay apparently 
```
    iptables auto-blocked IPs:
      x.x.x.x (unlimited timeout)
            [expired timeout, sending cleanup message]
```
Unlimited timeout. Hmm. So how can unlimited timeout?

**Comment by mrash** [comment]
I have reproduced this, will publish a fix soon.

**Comment by mrash** [comment]
This has been fixed in psad-2.4.4.
